web/client/epics/__tests__/styleeditor-test.js

```diff
@@ -466,141 +467,283 @@ describe('Test styleeditor epics', () => {
             state);
     });
 
-    it('test createStyleEpic', (done) => {
-        const state = {
-            layers: {
-                flat: [
-                    {
-                        id: 'layerId',
-                        name: 'layerName',
-                        url: 'base/web/client/test-resources/geoserver/',
-                        describeFeatureType: {},
-                        style: 'test_style',
-                        layerFilter: {
-                            filters: [{id: INTERACTIVE_LEGEND_ID, "test": "test"}]
-                        },
-                        enableInteractiveLegend: true
+    describe("tests for createStyleEpic", () => {
+        beforeEach(done => {
+            setConfigProp('miscSettings', { experimentalInteractiveLegend: true });
+            setTimeout(done);
+        });
+        afterEach(done => {
+            setConfigProp('miscSettings', { });
+            setTimeout(done);
+        });
```

Comment:
Why now this tests are including the beforeEach and afterEach

Reply:
I just did it as setting **experimentalInteractiveLegend** with true only needed for tests **createStyleEpic**